preload/link-header-preload-imagesrcset.html

```diff
@@ -9,33 +9,39 @@
     setup({explicit_done: true});
 
     var iterations = 0;
+    var expectation = new Array(10).fill(0);
+    if (window.devicePixelRatio == 1) {
+        expectation[0] = expectation[5] = expectation[8] = 1;
+    } else if (window.devicePixelRatio == 2) {
```

Comment:
You're right that the initial assumption of DPR==1 is wrong, but assuming it's either 1 or 2 is not great either. Maybe we can have the expectations for DPR<1.5 and DPR>1.5? (or something along those lines)

Reply:
Sounds reasonable! I updated the test.